frameworks/g-cloud-12/questions/services/metricsHostingHow.yml

```diff
@@ -25,4 +25,4 @@ options:
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Select a type of metric.
```

Comment:
This needs changing - we're after a reporting method, not a metric type. I'll fix